src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React, { type Element } from 'react';
```

Comment:
I thought our lint rule would require a space between `@flow` and the first import statement. 

I am not saying you should fix it, I am just curious as to why it is not reporting error here 🤔

Reply:
I disabled ESLint for these fixtures in `.eslintignore`

Reply:
That would explain it 💡 any reason for ignoring these files?

Reply:
Ok, I see, it is necessary, if not the tests will fail from the eslint formatting.